tutorials/how-to-migrate-boost-v1-to-boost-v2.md

```diff
@@ -34,6 +34,35 @@ We tested several databases and found that YugabyteDB is best suited to the inde
 * it has great documentation
 * once set up, it can be managed through a web UI
 
+### Connecting multiple boost instances to a single LID
+
+It is possible to connect multiple boostd instances to a single LID instance.
+In this scenario, each boostd instance still stores data to a single miner.
+eg boostd A stores data to miner A, boostd B stores data to miner B.
+However each boostd instance saves retrieval indexes in a single, shared LID instance.
+
+![LID Flow](assets/LID-multi-store.png)
+
+For retrieval, each boostd instance can query the shared LID
+instance (to find out which miner has the data) and retrieve data from any miner
+in the cluster.
+
+![LID Flow](assets/LID-multi-fetch.png)
+
+booster-bitswap and booster-http can also be configured to query the shared LID
+instance, and retrieve data from any miner in the cluster.
+
+![LID Flow](assets/LID-multi.png)
+
+{% hint style="warning" %}
+If you are deploying multiple boost instances with a single LID
+instance you will need to set up the networking so that each boost instance
```

Comment:
```suggestion
instance you will need to set up the networking so that each boostd instance
```

Reply:
I used "boost instance" here rather than boostd, because it could be booster-http or booster-bitswap also

Reply:
That makes sense